vispy/scene/cameras/base_camera.py

```diff
@@ -108,6 +109,19 @@ def depth_value(self, value):
             raise ValueError('depth value must be positive')
         self._depth_value = value
 
+    @property
```

Comment:
@campagnola are you okay with this, or should we leave it out? I'm okay either way.